modules/core/src/controllers/first-person-controller.js

```diff
@@ -95,7 +94,10 @@ class FirstPersonState extends ViewState {
    */
   pan({pos, startPos}) {
     const startPanEventPosition = this._interactiveState.startPanEventPosition || startPos;
-    assert(startPanEventPosition, '`startPanEventPosition` props is required');
+
+    if (!startPanEventPosition) {
```

Comment:
When do these conditions actually happen? Add a comment to give reader a clue?

Reply:
This happens when the mouse started dragging outside of this viewport, but drags over it.
I plan to add explicit flags to `interactionState` such as `isMoving` `isRotating` in a follow up PR. It will make the code easier to read.